data/json/items/vehicle/rotor.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "id": "heavy_duty_military_rotor",
```

Comment:
And this

Reply:
What could a rotor looks_like currently?

Reply:
ah nm, I went with a wind turbine :D